src/graphics/reproject-texture.js

```diff
@@ -98,11 +110,12 @@ function reprojectTexture(source, target, options = {}) {
         "#define SOURCE_FUNC " + sourceFunc + "\n" +
         "#define TARGET_FUNC " + targetFunc + "\n" +
         "#define NUM_SAMPLES " + numSamples + "\n" +
-        "#define NUM_SAMPLES_SQRT " + Math.round(Math.sqrt(numSamples)).toFixed(1) + "\n\n" +
+        "#define NUM_SAMPLES_SQRT " + Math.round(Math.sqrt(numSamples)).toFixed(1) + "\n" +
```

Comment:
this block should be converted to a string literal template

Reply:
Agree. I thought about it, but was too lazy at the time :)

Reply:
done